### PR TITLE
Adding "Big text block" and "testimonial" BEAN types

### PR DESCRIPTION
**Pull request by jbickar**
See BASIC-919
See BASIC-914

### CODE REVIEW

File: stanford_bean_types.bean.inc [hunk 1]
```diff
@@ -22,6 +22,15 @@ function stanford_bean_types_bean_admin_ui_types() {
   $bean_type = new stdClass();
   $bean_type->disabled = FALSE; /* Edit this to true to make a default bean_type disabled initially */
   $bean_type->api_version = 5;
+  $bean_type->name = 'stanford_big_text_block';
+  $bean_type->label = 'stanford_big_text_block';
+  $bean_type->options = '';
+  $bean_type->description = '';
```
Comment (sherakama) [inline stanford_bean_types.bean.inc:28]: This is a bit nit pickey but please add a description about what this block is and does.

File: stanford_bean_types.bean.inc [hunk 2]
```diff
@@ -56,5 +65,14 @@ You can insert files and images into your body content. There is no header image
   $bean_type->description = 'Add your social media links (e.g., Facebook, Twitter, YouTube, etc.)';
   $export['stanford_social_media_connect'] = $bean_type;
 
+  $bean_type = new stdClass();
+  $bean_type->disabled = FALSE; /* Edit this to true to make a default bean_type disabled initially */
+  $bean_type->api_version = 5;
+  $bean_type->name = 'stanford_testimonial_block';
+  $bean_type->label = 'stanford_testimonial_block';
+  $bean_type->options = '';
+  $bean_type->description = '';
```
Comment (sherakama) [inline stanford_bean_types.bean.inc]: This is a bit nit pickey but please add a description about what this block is and does.

Comment (jbickar) [inline stanford_bean_types.bean.inc]: I blame @linneawilliams

Comment (sherakama) [inline stanford_bean_types.bean.inc]: I know not how to respond to this.

File: stanford_bean_types.bean.inc (hunk 1 above)
Comment (linneawilliams) [inline stanford_bean_types.bean.inc:28]: Testimonial BEANs are used to feature a headshot, quote, and attribution.

**Pull request merged**